pkg/minikube/cluster/cluster.go

```diff
@@ -78,6 +78,10 @@ func StartHost(api libmachine.API, config cfg.MachineConfig) (*host.Host, error)
 		return nil, errors.Wrap(err, "Error loading existing host. Please try running [minikube delete], then run [minikube start] again.")
 	}
 
+	if h.Driver.DriverName() != "none" && h.Driver.DriverName() != config.VMDriver {
```

Comment:
Would we lose anything if "none" wasn't a special case here?

Reply:
Thanks for the review. I will remove the "none check" as none driver is also not changed in the restart process.